File: Back/src/ProEventos.Domain/Speaker.cs
using System.Collections.Generic;
namespace ProEventos.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Curriculum { get; set; }
        public string ImagemURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakersEvents { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/Speaker.ts
import { SocialMedia } from "./SocialMedia";
import { Event } from "./Event";

export interface Speaker {
  id : number;
  name : string;
  curriculum : string;
  imagemURL : string;
  phone : string;
  email : string;
  socialMedias : SocialMedia[];
  speakersEvents : Event[];
}
